### ISSUE TITLE
auto_commit in save_to_database and save_book_to_database to allow rollback

### ISSUE THREAD

**Issue by crbertoldo** [pyexcel-webwares/Flask-Excel#47]
Hello,

I suggest telling about the **auto_commit** parameter in the documentation for **save_to_database** and **save_book_to_database** methods. In my need, when uploading a big table, if any row goes wrong (e.g.: foreign key constraint fails) I'd like to roll back the whole process. With auto_commit=False (default, omitted) I cannot, but when passing it as True and then committing the db.session, if any row goes wrong, an exception can be raised and then I perform the rollback, like below:

```       
try:
    request.save_to_database(
        field_name='file',
        session=db.session,
        table=model,
        initializer=init_func,
        auto_commit=False)
    db.session.commit()
except:
    db.session.rollback()
```

**Comment by chfw** [comment]
Are you suggesting a documentation update? What you are doing is agreeable and this library did not prevent you from doing it.

**Comment by crbertoldo** [comment]
@chfw, yes, a documentation update, please. And also to inform here who needs this.

**Comment by chfw** [comment]
Sure! Thanks for using this library. I will get it update over the weekend.
